### ISSUE TITLE
Broken docs link

### ISSUE THREAD

**Issue by Vishwas-Adiga** [Kodular/Kodular-Creator-Issue-Tracker#20]
https://docs.kodular.io/blocks/colors/#make

**Comment by yanquisalexander** [comment]
https://github.com/Kodular/Kodular-Creator-Issue-Tracker/issues/8

Related? 🤔

**Comment by Vishwas-Adiga** [comment]
No. The page lists an RGB chart which redirects [here](http://www.tayloredmktg.com/rgb/). That page is no longer active.

**Comment by pavi2410** [comment]
@Vishwas-Adiga I can't reproduce this now. Can you try again?
